src/app/clusters/mode-select-server/static-supported-modes-manager.cpp

```diff
@@ -23,35 +23,42 @@ Structs::ModeOptionStruct::Type buildModeOptionStruct(const char * label, uint8_
 }
 } // namespace
 
-const Structs::ModeOptionStruct::Type StaticSupportedModesManager::blackOption      = buildModeOptionStruct("Black", 0, 0);
-const Structs::ModeOptionStruct::Type StaticSupportedModesManager::cappuccinoOption = buildModeOptionStruct("Cappuccino", 4, 0);
-const Structs::ModeOptionStruct::Type StaticSupportedModesManager::espressoOption   = buildModeOptionStruct("Espresso", 7, 0);
-storage_value_type StaticSupportedModesManager::coffeeOptions[] = { &blackOption, &cappuccinoOption, &espressoOption };
-const Span<storage_value_type> StaticSupportedModesManager::coffeeOptionsSpan =
-    Span<storage_value_type>(StaticSupportedModesManager::coffeeOptions);
-const map<EndpointId, Span<storage_value_type>> StaticSupportedModesManager::optionsByEndpoints = {
-    { 1, StaticSupportedModesManager::coffeeOptionsSpan }
-};
+// TODO: Configure your options for each endpoint
+storage_value_type StaticSupportedModesManager::coffeeOptions[] = { buildModeOptionStruct("Black", 0, 0),
+                                                                    buildModeOptionStruct("Cappuccino", 4, 0),
+                                                                    buildModeOptionStruct("Espresso", 7, 0) };
+const StaticSupportedModesManager::EndpointSpanPair
+    StaticSupportedModesManager::supportedOptionsByEndpoints[ENDPOINTS_WITH_MODES] = {
+        EndpointSpanPair(1, Span<storage_value_type>(StaticSupportedModesManager::coffeeOptions)) // Options for Endpoint 1
+    };
```

Comment:
Why are these fixed examples in core SDK code (e.g. `src/app`) rather than  in `examples/`